ch2/procfs_simple_intf/procfs_simple_intf.c

```diff
@@ -395,7 +427,11 @@ static void __exit procfs_simple_intf_cleanup(void)
 {
 	gdrvctx->power = 0;
 	remove_proc_subtree(OURMODNAME, NULL);
+#if LINUX_VERSION_CODE < KERNEL_VERSION(6, 5, 0)
```

Comment:
ditto to above comment

Reply:
Again thanks for the help!

Reply:
Welcome! Thanks @seantywork for the work, these updates!